modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java

```diff
@@ -62,6 +62,37 @@ public CommerceOrder addCommerceOrder(
 			userId, groupId, commerceAccountId, commerceCurrencyId);
 	}
 
+	@Override
+	public CommerceOrder addCommerceOrder(
+			long userId, long groupId, long commerceAccountId,
+			long commerceCurrencyId, long commerceOrderTypeId)
```

Comment:
Remove `long userId`, remote services must not allow anyone to pass in `userId`. We should use `getUserId()`.